### PR TITLE
Add visits table and additional Queries

### PR DESCRIPTION
**Pull request by tjay1760**
### Changes Made

- Created tables `vets`, `specializations`, and `visits` as specified.
image

- Inserted data into the `vets`, `specializations`, and `visits` tables.
image
image
image

### Files Changed

- Created `schema.sql` file containing the table creation scripts.
- Created `data.sql` file containing the data insertion scripts.
- Created `queries.sql` file containing the required queries.

### Query Results
Write queries to answer the following:
Who was the last animal seen by William Tatcher?
image

How many different animals did Stephanie Mendez see?
image

List all vets and their specialties, including vets with no specialties.
image

List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
image

What animal has the most visits to vets?
image

Who was Maisy Smith's first visit?
image

Details for most recent visit: animal information, vet information, and date of visit.
image

How many visits were with a vet that did not specialize in that animal's species?
image

What specialty should Maisy Smith consider getting? Look for the species she gets the most.
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -42,4 +42,92 @@ GROUP BY name, neutered;
...
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+SELECT vets.name as vet , (SELECT name FROM species WHERE id = specializations.species_id) as species FROM vets 
+LEFT JOIN specializations ON specializations.vet_id =vets.id;
+
+SELECT (SELECT name FROM animals WHERE id = visits.animal_id) AS animal 
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = ' Stephanie Mendez')
+AND visits.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
```
Comment (bobb-Rob) [inline queries.sql]: - [ ] Your query result as shown on your screenshot for this - `List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.` should show two results as highlighted below: 👇🏾 Please check your data or query command and correct accordingly.

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @tjay1760, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ All linters checks are ok
✅ The correct workflow is used. 
✅ Very descriptive Pull Request(PR)
✅ Your work is well documented.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (tjay1760) [reply queries.sql]: I have Updated the query 
image

**Review by Reem-lab — APPROVED** [review]
# Hi @tjay1760, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descrtipive PR⭐
- [ ] Kudos for working on the functionality for the queries🙏🏻




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**